controllers/MainController.php

```diff
@@ -7,6 +7,7 @@
 
 class MainController extends Controller
 {
+
```

Comment:
@luke- are there any plans for using this controller or its view, if not it should be okay to delete yes?